### ISSUE TITLE
blur / blackout screen during ads

### ISSUE THREAD

**Issue by pandrewhk** [octalmage/lull#9]

**Comment by octalmage** [comment]
I experimented with this and I wasn't a fan. I like seeing the ad so I know it's still playing. I'll add it in and make it off by default!

**Comment by infofarmer** [comment]
Hmm. Maybe just something to tone it down a notch? Black and white, lower brightness, etc?

**Comment by octalmage** [comment]
Yeah I think maybe I'll overlay a transparent black box, so it's dimmed. I'll play with it and post the results here.

**Comment by octalmage** [comment]
I had another app that I personally used that would overlay a YouTube playlist, so the entertainment was continuous. This might be something I add in later.

**Comment by infofarmer** [comment]
Gave up and upgraded to the ad–less plan a few weeks ago 🙂

**Comment by octalmage** [comment]
@infofarmer Hows that been? Is it worth it? I heard only like half of the videos are ad free or something like that.

**Comment by pandrewhk** [comment]
Actually I started Hulu, just because of Criterion movies, which are ad–free either way. Got sucked into a few shows — that's when I upgraded. Since then, downgraded again then cancelled and switched to Netflix 🙂 So — that's that.
